internal/hud/server/server.go

```diff
@@ -68,20 +71,23 @@ func ProvideHeadsUpServer(
 	store *store.Store,
 	assetServer assets.Server,
 	analytics *tiltanalytics.TiltAnalytics,
+	metrics *metrics.ModeController,
 	uploader cloud.SnapshotUploader) (*HeadsUpServer, error) {
 	r := mux.NewRouter().UseEncodedPath()
 	s := &HeadsUpServer{
 		ctx:      ctx,
 		store:    store,
 		router:   r,
 		a:        analytics,
+		metrics:  metrics,
 		uploader: uploader,
 	}
 
 	r.HandleFunc("/api/view", s.ViewJSON)
 	r.HandleFunc("/api/dump/engine", s.DumpEngineJSON)
 	r.HandleFunc("/api/analytics", s.HandleAnalytics)
 	r.HandleFunc("/api/analytics_opt", s.HandleAnalyticsOpt)
+	r.HandleFunc("/api/metrics_opt", s.HandleMetricsOpt)
```

Comment:
somewhere (idk if in `model` or where we define `MetricsOptAction` vs. `AnalyticsOptAction`) it'd be nice to have a comment differentiating Metrics from Analytics.

Reply:
done